packages/store/src/module.ts

```diff
@@ -87,11 +92,13 @@ export class NgxsFeatureModule {
       stateOperations.setState({ ...cur, ...results.defaults });
     }
 
-    stateOperations.dispatch(new UpdateState()).subscribe(() => {
-      if (results) {
-        factory.invokeInit(results.states);
-      }
-    });
+    // dispatch the update action and invoke init and bootstrap functions after
+    internalStateOperations.dispatchActionAndInvokeLifecyleHooks(
```

Comment:
Rename and do as here
https://github.com/ngxs/store/blob/1ccc958b920e13bbd46d6f84976fb56cd00657d0/packages/store/src/module.ts#L40